modules/nw-ingress-controller-endpoint-publishing-strategies.adoc

```diff
@@ -33,3 +34,23 @@ For more information, see the link:https://kubernetes.io/docs/concepts/services-
 The `HostNetwork` endpoint publishing strategy publishes the Ingress Controller on node ports where the Ingress Controller is deployed.
 
 An Ingress Controller with the `HostNetwork` endpoint publishing strategy can have only one pod replica per node. If you want _n_ replicas, you must use at least _n_ nodes where those replicas can be scheduled. Because each pod replica requests ports `80` and `443` on the node host where it is scheduled, a replica cannot be scheduled to a node if another pod on the same node is using those ports.
+
+`HostNetwork` has a `hostNetwork` field with the following default values for the optional binding ports: `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`. By specifying binding ports for your network, you can deploy multiple Ingress Controllers on the same node for the `HostNetwork` strategy.
```

Comment:
rather than:  "By specifying binding ports for your network, you can deploy multiple Ingress Controllers on the same nodefor the `HostNetwork` strategy."
I'd like to see: By specifying different binding ports for your network, it is possib le deploy multiple Ingress Controllers on the same node for the `HostNetwork` strategy."

Reply:
Thanks. Updated!